src/modules/referrals/components/ProjectOutgoingReferralPostingsTable.tsx

```diff
@@ -119,6 +119,8 @@ const ProjectOutgoingReferralPostingsTable: React.FC<Props> = ({
       filters={{ status: referralFilter }}
       defaultPageSize={15}
       paginationItemName='outgoing referral'
+      // If User has access to the receiving project, link to the Referral
```

Comment:
Maybe, move this comment above line 69.

Perhaps also consider adding a rowLinkTo for the referral, if the user has access?